src/Libm.jl

```diff
@@ -1,5 +1,26 @@
 module Libm
 
 # package code goes here
+include("erf.jl")
+
+
+
+function GET_HIGH_WORD(d::Float64)
+    bd = bits(d)
+    hi = bd[1:32]
+    hi = parse(UInt32,hi,2)
+    return hi
+end
+
+function SET_LOW_WORD(d::Float64,lo::UInt)
```

Comment:
```julia
SET_LOW_WORD(d::Float64, lo::UInt32) = reinterpret(Float64, reinterpret(UInt64, d) & 0xFFFFFFFF00000000 | lo)
```
should be faster.

Also note that in general to get faster code you should avoid changing the type of the variable throughout the function as it makes Julia generating type-agnostic (much slower) code:

```
julia> @code_warntype SET_LOW_WORD(1.0, UInt(5))
Variables:
  #self#::#SET_LOW_WORD
  d@_2::Float64
  lo@_3::UInt64
  db::String
  lb::String
  d@_6::Any
  lo@_7::Any

Body:
  begin 
      lo@_7::Any = lo@_3::UInt64
      d@_6::Any = d@_2::Float64
      db::String = $(Expr(:invoke, LambdaInfo for bin(::UInt64, ::Int64, ::Bool), :(Base.bin), :((Base.box)(UInt64,d@_6::Float64)), 64, false)) # line 3:
      lo@_7::Any = (Base.box)(UInt32,(Base.checked_trunc_uint)(UInt32,lo@_7::UInt64)) # line 4:
      lb::String = $(Expr(:invoke, LambdaInfo for bin(::UInt32, ::Int64, ::Bool), :(Base.bin), :((Base.box)(UInt32,lo@_7::UInt32)), 32, false)) # line 5:
      SSAValue(0) = $(Expr(:invoke, LambdaInfo for getindex(::String, ::UnitRange{Int64}), :(Main.getindex), :(db), :($(Expr(:new, UnitRange{Int64}, 1, :((Base.select_value)((Base.sle_int)(1,32)::Bool,32,(Base.box)(Int64,(Base.sub_int)(1,1)))::Int64))))))
      d@_6::Any = $(Expr(:invoke, LambdaInfo for string(::String, ::String, ::Vararg{String,N}), :(Base.string), SSAValue(0), :(lb))) # line 6:
      d@_6::Any = $(Expr(:invoke, LambdaInfo for parse(::Type{Int64}, ::String, ::Int64), :(Main.parse), :(Main.Int), :(d@_6::String), 2)) # line 7:
      d@_6::Any = (Base.box)(Float64,d@_6::Int64) # line 8:
      return d@_6::Float64
  end::Float64
```
(`::Any` signifies type inference problem)